src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -450,26 +453,26 @@ def launch_fluent(
     case_filepath: str = None,
     case_data_filepath: str = None,
     lightweight_mode: bool = False,
-    mode: Union[LaunchModes, str, None] = None,
+    mode: Union[LaunchMode, str, None] = None,
     server_info_filepath: str = None,
     password: str = None,
     py: bool = None,
     cwd: str = None,
     topy: Union[str, list] = None,
     **kwargs,
-) -> Union[Meshing, PureMeshing, Solver]:
+) -> Union[Meshing, PureMeshing, Solver, SolverIcing]:
     """Launch Fluent locally in server mode or connect to a running Fluent
     server instance.
 
     Parameters
     ----------
     product_version : str, optional
-        Version of Fluent to use in the numeric format (such as ``"23.1.0"``
-        for 2023 R1). The default is ``None``, in which case the active version
-        or latest installed version is used.
+        Select an installed version of ANSYS. The string must match the
+        documented version format in the FluentVersion class. The default is
```

Comment:
The version format is not part of docstring in the FluentVersion class. So, if the user is in Jupyter/IDE they will not know the version format unless they look up the doc website.